src/components/home/partners/Partners.jsx

```diff
@@ -2,8 +2,10 @@ import Typography from '@mui/material/Typography';
 import PrimaryPartner from './PrimaryPartner';
 import SecondaryPartners from './SecondaryPartners';
 import CallToAction from './CallToAction';
+// Other Library Imports
+import { PropTypes } from 'prop-types';
 
-const Partners = () => {
+const Partners = (handleContactForm) => {
```

Comment:
Now the `ContactFormModal` got moved into the `CallToAction` section/component...

so no need to pass `handleContactForm` to `CallToAction` here on line 23, and no need to "receive" `handleContactForm` on line 8 (which `Home` wasn't passing it anyway & it wasn't destructured.) Then, you'll also be able to delete lines 28-30.

But now won't be able to reuse this easily for the `footer`, can deal with that at another time like you said.

Reply:
Fixed